WordPress/Classes/Stores/PluginStore.swift

```diff
@@ -19,25 +22,36 @@ enum PluginAction: Action {
 
 enum PluginQuery {
     case all(site: JetpackSiteRef)
+    case featured(site: JetpackSiteRef)
```

Comment:
I'm not sure I understand why `featured` has a `site` reference. If it's just so you can ensure you get the installed plugins to show status, then it should also be in `feed` (and maybe `directoryEntry`), right?